### DISCUSSION TITLE
cant block a "specific Client " on my internal network

### DISCUSSION THREAD

**Discussion by baloo666** [AdguardTeam/AdGuardHome#7409] [General]
Hello everyone,
i'm using adguard home via synology docker
v0.107.53
I searched the forums and made many attempts to block a specific user on my internal network but without success
I understood that the blocking option is carried out by "TAGS" with a specific name that I give to the computer, but in practice it is not possible to add new TAGS (there is no option or button that allows this), but only to use the existing TAGS and associate them with my computers
I also tried this option but it didn't block my user
An example of a laptop that I wanted to block and these are the attempts I made following searches in the forums:
||[www.babobotanicals.com](http://www.babobotanicals.com/)^$ctag='device_laptop'
|[babobotanicals.com](http://babobotanicals.com/)^$client='Lenovo Laptop'

When I blocked specific user  through "for this client only" this is what the software added by itself
 ||[play.google.com](http://play.google.com/)^$client='HTPC'
But even in this case the user was not blocked!

I would be very happy if you could explain to me how I block a single user

**Comment by Dynamic5912** [comment]
What device is it? Make, model etc.

**Reply by baloo666** [reply]
its synology NAS  model DS920+ based on docker

Is there an "add new tags" button that I don't see?
I tried almost everything...

**Reply by Dynamic5912** [reply]
> its synology NAS  model DS920+ based on docker
> 
> 
> 
> Is there an "add new tags" button that I don't see?
> 
> I tried almost everything...

The device you're trying to block...

**Reply by baloo666** [reply]
not device ,websites for specific users 
i can block websites but all my other computers in my network will block to 
i looked at your  "AdGuard-Home-Lists" but in your  Personal Blocklist i didnt saw that you blocked any websites for a specific host  just for your all  your network 
am i right?

**Reply by Dynamic5912** [reply]
Yes. Hence my question.

What is the make and model etc. for the device you are trying to block the access for?

**Reply by baloo666** [reply]
Lenovo laptop - ideapad5 for example

**Reply by Dynamic5912** [reply]
> Lenovo laptop - ideapad5 for example  

What is the DNS pointing to on the laptop and on the router?

When you block a website for the client, what does the Query Log in AGH show?

**Reply by baloo666** [reply]
the laptop dns pointing to my synology nas server (behaving as a DNS server) my router pointing to my synology nas server to according to the log file there is a lot of errors but As far as I understand, no one is related to my problem
Can you point me in the direction of adding a TAG to a user
Because in my opinion this is my main problem that I just don't understand how or where to do it
When I block the whole network there is no problem so it doesn't seem to be the problem otherwise no computer would have been blocked
my version is v0.107.53 maybe there is a problem with her?

**Reply by Dynamic5912** [reply]
Have you added the device as a client into AGH and set the tag/s against that device?

**Reply by baloo666** [reply]
yes , and yet nothing happened 
but my question is can i add some new TAGS or its impossible? 
maybe my syntax is incorrect when i add it manualy?
can you give me (from your adguard configuraton) the right syntax for block a specific user??

**Reply by Dynamic5912** [reply]
> yes , and yet nothing happened 
> 
> but my question is can i add some new TAGS or its impossible? 
> 
> maybe my syntax is incorrect when i add it manualy?
> 
> can you give me (from your adguard configuraton) the right syntax for block a specific user??

If you want to block something based on Tags, just click "block for current device" in the query log for the domain you don't want it to access

**Reply by baloo666** [reply]
As I wrote to you before, I did it and it still didn't block the specific user I wanted
When I blocked specific user through the log "for this client only" this is what the software added by itself
||[play.google.com](http://play.google.com/)^$client='HTPC'
But even in this case the user was not blocked!

**Reply by Dynamic5912** [reply]
> As I wrote to you before, I did it and it still didn't block the specific user I wanted
> 
> When I blocked specific user through the log "for this client only" this is what the software added by itself
> 
> ||[play.google.com](http://play.google.com/)^$client='HTPC'
> 
> But even in this case the user was not blocked!

You're blocking an individual device, not a user.

How do you know it still works?

Why is Google Play being blocked on a Lenovo laptop?

**Reply by baloo666** [reply]
my mistake I meant the device not the user but it still doesn't matter because it doesn't work
Google Play is just an attempt to block a site, it doesn't matter
No site has been blocked for me

**Reply by Dynamic5912** [reply]
> my mistake I meant the device not the user but it still doesn't matter because it doesn't work
> 
> Google Play is just an attempt to block a site, it doesn't matter
> 
> No site has been blocked for me
> 
> 

And once again - what does the AGH query log show for that site after it's been blocked and the device tries to access it?
